commands/download.go

```diff
@@ -0,0 +1,72 @@
+package commands
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	faasCmd.AddCommand(downloadCmd)
+}
+
+var downloadCmd = &cobra.Command{
+	Use:     `download`,
+	Short:   "Download binaries",
+	Long:    "Download binaries",
+	Example: "faas-cli download",
+	Run:     runFunc,
+}
+
+func runFunc(cmd *cobra.Command, args []string) {
+	cmd.Help()
+	if len(args) == 0 {
+		fmt.Printf("You can download: %s\n", strings.TrimRight(strings.Join(getDownloads(), ", "), ", "))
+	}
+}
+
+func getDownloads() []string {
+	return []string{"kubeseal"}
+}
+
+func downloadBinary(url string) error {
```

Comment:
The implementation seems too specific to kubeseal, i think it would be easiest to keep these details in `downloadKubesealBinary` method.  Or it needs to be made more generic, e.g. passing the target name as a string`func downloadBinary(url string, name string) error`

Reply:
I see your point and I'm also OK with specific until we have two use-cases at which point it would make a good case to refactor the private function.